### ISSUE TITLE
Feature Request: Clock options

### ISSUE THREAD

**Issue by ddantzler** [lzanita09/NotificationPeekPort#16]
Configuration option to use current clock type or PA Peek clock type (IE: Mimic status bar clock).

**Comment by lalodroid** [comment]
I had same idea... maybe would be better use the PA Peek clock type by default.

**Comment by lzanita09** [comment]
I'll try, PA Peek has a different implementation, but I think the Activity can also do that, just a different layout.

**Comment by ddantzler** [comment]
Awesome. As mentioned, would prefer it as an option, not necessarily a
replacement.

**Comment by lzanita09** [comment]
Yes, that'll be an option. I'm thinking about how to make the option intuitive, maybe a preview.

**Comment by ddantzler** [comment]
Awesome. Take your time on this, it's aesthetic and of course you can't please everyone with that type of thing. Obviously, functionality is paramount. I've tested basically every notification application for
Android, and I am digging this the best so far. Minimal, unobtrusive, and efficient. It seems like a feature that should be baked in to AOSP. You're doing great! I've seen developers take the path of trying to please all of the users on look and feel too far, and it's a job that can never be completed. By OPing this, I was just putting it on the board for future thought.

**Comment by lalodroid** [comment]
as ddantzler said. Functionality should be the target. you are doing a excellent work with this app, please do not add a lot of features and options, keep it minimalist and simple. 
off topic: will you put the app in playstore? what about PA team?

**Comment by lzanita09** [comment]
I'll try to make the grouping work first. 

I prefer to publish it on Google Play, but I didn't receive any comment from PA, I'm not sure if I can do that.

**Comment by ddantzler** [comment]
Is this truly a port of Peek, or a "reverse engineering" of Peek
functionality. I'm no lawyer, but in my mind PA is an open source project,
and either way it should be okay if you don't intend to profit on it and
keep it open source (except via donations). However, you may not want to
spite the folks, and I can understand that.

**Comment by lzanita09** [comment]
It is a port of Peek, the source code of the original Peek can be found in PA's open source repository. I think you're right, I'm not planning to get profit from it and I have published the code here.
